packages/timeline-state-resolver/src/devices/atem.ts

```diff
@@ -468,4 +479,9 @@ export class AtemDevice extends DeviceWithState<DeviceState, DeviceOptionsAtemIn
 	private _connectionChanged() {
 		this.emit('connectionChanged', this.getStatus())
 	}
+	private _isAssignableToNextStyle(transition: AtemTransitionStyle | undefined) {
+		return (
```

Comment:
The paranthesis are not necessary.
Also, would be nice that we specified the return type of our methods. We are writing TypeScript after all :)

Reply:
The parentheses are necessary according to the linter.
We are using TypeScript and I'm using its feature called type Inference.

Reply:
The added return type is for our, and future developers, benefit. It's so you don't have to read the entire implementation of a method just to find out what type it's returning.

Take this scenario:
```
function A(): void {
  const b = B();
  // Do stuff with b
}
```
(For this, ignore the fact that they didn't write a return type on the variable `b`, something that is happening a lot in Sofie)

I have no idea what type `b` is and to figure it out I have to look at the method declaration of `B()`. Only issue is, the return type on the method declaration has been omitted.
So now, when I am actually trying to read and understand `A()`, I have to instead go and read understand `B()` first because I want to know the type of `b`.

It's stuff like this, that might seem trivial, that adds to how long it takes to read and understand a code snippet. And in the end, readable code is all about how long it takes to read and understand.

Sidenote: The above mentioned scenario is something I encounter quite often in Sofie.